docs/usage.md

```diff
@@ -105,7 +105,7 @@ class Product
      * 
      * @Vich\UploadableField(mapping="product_image", fileNameProperty="imageName", size="imageSize")
      * 
-     * @var File
+     * @var ?File
```

Comment:
Can we use `@var File|null` instead?

Reply:
Sure :)

I wanted to test the annotation proposed here https://github.com/symfony/symfony/pull/32107
 but it doesn't seem to be present in my symfony 5.0.2 project.